runtime/include/chpl-comm.h

```diff
@@ -82,14 +82,14 @@ extern const int chpl_heterogeneous;
 // before the request completes (after waiting on the returned handle)
 chpl_comm_nb_handle_t chpl_comm_get_nb(void* addr, c_nodeid_t node, void* raddr,
                                        size_t size, int32_t typeIndex,
-                                       int ln, c_string fn);
+                                       int ln, int32_t fn);
```

Comment:
Contrast this with my previous comment: here we went from (effectively) "const char*" to just plain "int32_t", no "const".